questions.md

```diff
@@ -0,0 +1,8 @@
+#Additional Questions
+
+* Here is where I can list any additional questions that I have
+
+1. I'm wondering what problems I may run into in the classroom
```

Comment:
@scday here are some resources for you to prepare your classroom for any potential roadblocks:
- A tutorial on merge conflicts in the classroom https://github.blog/2018-08-22-merge-conflicts-in-the-classroom/
- A talk on how 3 teachers made space to implement GitHub in their classrooms https://github.blog/2018-03-15-sigcse-real-world-tools/